docs/3_Scope.md

```diff
@@ -0,0 +1,8 @@
+# Scope
+This document outlines the various use cases that are in and out of scope for this specification.
+
+## Use Cases In Scope
```

Comment:
Mention In Scope Items as : 

1. Buying & Seller of Grocery
2. Buying & Seller of Electronic Items
3. uying & Seller of F&B